gt/main.c

```diff
@@ -2209,6 +2252,70 @@ l_update_gt_lua_states(lua_State *l)
 	return 0;
 }
 
+static void
+process_return_message(lua_State *l, struct dynamic_config *dy_conf,
```

Comment:
Although `l_update_gt_lua_states_incrementally()` is implemented here in file `gt/main.c`, it's the dynamic configuration block that calls and executes it. We have originally placed `l_update_gt_lua_states_incrementally()` here because it accesses `struct gt_config`, but with the addition of `process_return_message()`, the code is really about the dynamic configuration block. Thus, I propose to move `l_update_gt_lua_states_incrementally()` and `process_return_message()` to file `config/dynamic.c`.

Once this rearrangement is done, `MSG_MAX_LEN` doesn't need to be exported.